.github/ISSUE_TEMPLATE/1question.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Question
+about: Seeking help or have a question about usage?
+labels: question
+---
+
+<!-- We can't debug your app for you, but you can ask questions and we will try to answer them.
+
+You can also ask a question in the Node.js Slack community's #express-js channel:
+
+* Slack invite form: http://www.nodeslackers.com/invite
+* #express-js channel: https://node-js.slack.com/archives/C0L827VV4
+
+It is super important that you paste in samples of your code (please avoid screenshots of code)!
```

Comment:
```suggestion
It is super important that you paste in samples of your code (no screenshots of code, [use markdown](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks))!
```

I feel like we should auto close anything which is a screenshot of code. And by linking here we help educate folks who might not know.